data/domains/movies.yaml

```diff
@@ -1,6 +1,6 @@
 slot_names:
   TITLE:
-  GENRE:
+  GENRES:
   ACTOR:
-  KEYWORD:
+  KEYWORDS:
```

Comment:
These should be kept in singular: these are used when annotating dialogues and modeling preferences and there it's a single genre or keyword.

Reply:
Removed plurals.